chart/values.yaml

```diff
@@ -406,6 +406,7 @@ io_engine:
 etcd:
   # Configuration for etcd's localpv hostpath storage class.
   localpvScConfig:
+    enabled: true
```

Comment:
This needs `# -- ` style comment to generate documentation.

Reply:
Not sure if we want this to be a part of the doc.

Reply:
Okay..